presto-main/src/test/java/com/facebook/presto/cost/TestComparisonStatsCalculator.java

```diff
@@ -178,6 +178,11 @@ public void setUp()
         };
     }
 
+    private SymbolStatsEstimate updateNDV(SymbolStatsEstimate symbolStats, double delta)
```

Comment:
it is not `ndv` it is `dvc`

Reply:
we never abbreviate this to "dvc", do we?

Reply:
IMO that was a mistake. We never called things in the code like `numberOfDistinctValues` (at least not in this project), that makes me think that `ndv` is only known to our office. It might be confusing to reader. Anyway, we called that estimate distinctValuesCount so it is more dvc than ndv.

Reply:
I think NDV is a pretty standard abbreviation.

Reply:
To illustrate my point: https://en.wikipedia.org/wiki/NDV

I think using `dvc` as an acronym would be very confusing.